shared/domain/src/commonMain/kotlin/com/android254/shared/domain/repos/HomeRepo.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2024 DroidconKE
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.android254.shared.domain.repos
+
+import com.android254.shared.domain.models.HomeDetails
+import kotlinx.coroutines.flow.Flow
+
+interface HomeRepo {
+    fun fetchHomeDetails(): Flow<HomeDetails>
```

Comment:
Do you mind adding the data implementation with a mock implementation.

Reply:
No, let me add it... I presume that is in the data module,right?

Reply:
@chepsi check if I added as expected